src/main/resources/application.yaml

```
@@ -48,4 +48,9 @@ logging:
     region.jidogam: INFO
     org.springframework: INFO
     org.hibernate.sql: INFO
-    org.hibernate.type.descriptor.sql.BasicBinder: INFO
\ No newline at end of file
+    org.hibernate.type.descriptor.sql.BasicBinder: INFO
+
+api:
+  area:
+    service-id: ${AREA_SERVICE_ID:dev-service-id}
+    secret-key: ${AREA_SECRET_KEY:dev-secret-key}
```

```suggestion
    secret-key: ${AREA_SECRET_KEY:dev-secret-key}

```

맨 아래 빈 줄이 없어서 크게 상관없지만 추가하면 좋을 것 같습니다.